### ISSUE TITLE
I may have two instances of Atom installed on my machine, how to remove? could it affect my primary Atom?

### ISSUE THREAD

**Issue by kfili** [ga-wdi-boston/game-project#297]
when i right-click to open a file in finder it shows seems that I have two atoms, or two versions of atom. Should I have two versions of atom installed?
Atom (1.11.2)
Atom 2 (1.11.1)

They open as seperate windows and as separate applications in the dock.
how do I uninstall Atom 2(I only see one Atom in the applications folder)? Could i mess my primary atom up some how if I uninstall atom 2?

**Comment by RealWeeks** [comment]
Goto applications and drag it into the trash.

Then do a finder search and drag the other one to the trash.

Reinstall Atom and make sure you move it to the applications folder.

**Comment by RealWeeks** [comment]
@kfili Any follow up on this?

**Comment by kfili** [comment]
I did find a file called Atom 2 in my downloads folder, moved it to trash, made no difference. I have one Atom in the applications folder. 

Another thing I noticed, when I right click on a file -> open with, it lists both atoms as versions 1.11.2 whereas before Atom 2 was 1.11.1
no idea why.

It all seems weird, it may having something to do with installing a new hard drive, and backing up from time machine. However, since its not causing any immediate issues and me being pressed for time im going to avoid reinstalling for now. Thanks!
